test/test_browser.py

```diff
@@ -2618,6 +2618,18 @@ def test_html5_webgl_create_context2(self):
   def test_html5_special_event_targets(self):
     self.btest_exit('html5_special_event_targets.cpp', args=['-lGL'])
 
+  @parameterized({
+    '': ([],),
+    'offscreen_no_pthread': (['-sOFFSCREENCANVAS_SUPPORT'],),
+    'offscreen_pthread': (['-sOFFSCREENCANVAS_SUPPORT', '-sPROXY_TO_PTHREAD', '-pthread'],),
+  })
+  @requires_graphics_hardware
+  # Verify bug https://github.com/emscripten-core/emscripten/issues/22942: findCanvasEventTarget doesn't use specialHTMLTargets
+  def test_html5_special_canvas_event_targets(self, args):
```

Comment:
You can do `self.emcc_args += ['-lGL', '-sDISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR=1'] + args` here to avoid duplication on the following 3 lines

Reply:
Cool, good to know